localstack/aws/forwarder.py

```diff
@@ -145,9 +146,14 @@ def create_aws_request_context(
     service = load_service(service_name)
     operation = service.operation_model(action)
 
-    # we re-use botocore internals here to serialize the HTTP request, but don't send it
+    # we re-use botocore internals here to serialize the HTTP request,
+    # but deactivate validation (validation errors should be handled by the backend)
+    # and don't send it yet
     client = aws_stack.connect_to_service(
-        service_name, endpoint_url=endpoint_url, region_name=region
+        service_name,
+        endpoint_url=endpoint_url,
+        region_name=region,
+        config=BotoConfig(parameter_validation=False),
```

Comment:
nit: should we maybe re-use the `BotoConfig` object? it looks like it's relatively expensive to create, but doesn't seem to carry any per-request state.